atd-vzv/src/views/summary/CrashesByPopulation.js

```diff
@@ -0,0 +1,173 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import styled from "styled-components";
+import { Bar } from "react-chartjs-2";
+import { Container, Row, Col } from "reactstrap";
+
+import CrashTypeSelector from "./Components/CrashTypeSelector";
+import InfoPopover from "../../Components/Popover/InfoPopover";
+import { popoverConfig } from "../../Components/Popover/popoverConfig";
+import { crashEndpointUrl } from "./queries/socrataQueries";
+import { dataStartDate, fiveYearAvgEndDate } from "../../constants/time";
+import { colors } from "../../constants/colors";
+
+const CrashesByPopulation = () => {
+  const [crashType, setCrashType] = useState(null);
+  const [chartData, setChartData] = useState({});
+
+  const url = `${crashEndpointUrl}?$query=`;
+
+  useEffect(() => {
+    const dateCondition = `crash_date BETWEEN '${dataStartDate.format(
+      "YYYY-MM-DD"
+    )}' and '${fiveYearAvgEndDate}'`;
+    const queryGroupAndOrder = `GROUP BY year ORDER BY year`;
+
+    const queries = {
+      fatalities: `SELECT date_extract_y(crash_date) as year, sum(death_cnt) as total 
+                   WHERE death_cnt > 0 AND ${dateCondition} ${queryGroupAndOrder}`,
+      fatalitiesAndSeriousInjuries: `SELECT date_extract_y(crash_date) as year, sum(death_cnt) + sum(sus_serious_injry_cnt) as total 
+                                     WHERE (death_cnt > 0 OR sus_serious_injry_cnt > 0) AND ${dateCondition} ${queryGroupAndOrder}`,
+      seriousInjuries: `SELECT date_extract_y(crash_date) as year, sum(sus_serious_injry_cnt) as total 
+                        WHERE sus_serious_injry_cnt > 0 AND ${dateCondition} ${queryGroupAndOrder}`,
+    };
+
+    const chartConfig = {
+      barOne: { color: colors.viridis1Of6Highest, population: 913917 },
+      barTwo: { color: colors.viridis1Of6Highest, population: 937065 },
+      barThree: { color: colors.viridis1Of6Highest, population: 955094 },
+      barFour: { color: colors.viridis1Of6Highest, population: 972499 },
```

Comment:
These threw me off a bit, since they are not all being used shouldn't they be removed?